### DISCUSSION TITLE
Being able to open links in a new tab

### DISCUSSION THREAD

**Discussion by minifisk** [markdoc/markdoc#498] [Q&A] [answered]
That would be awesome!

**Comment by mfix-stripe** [comment, marked as answer]
You can handle this by creating a custom `link` [`Node`](https://markdoc.dev/docs/nodes) 👍

**Comment by rpaul-stripe** [comment]
We've considered adding some syntax for supporting attributes on links and images, which would make this possible with standard link syntax plus an attribute. You can see the open issue here with some thoughts on syntax: https://github.com/markdoc/markdoc/issues/156#issuecomment-1580979257 If we were to implement attributes in links, I imagine that opening links in new tabs is something we'd want to support in the default schema ootb. I'm not sure yet if we are going to move forward with adding those features.
